torch/distributed/checkpoint/state_dict.py

```diff
@@ -134,6 +134,7 @@ class StateDictOptions:
     strict: bool = True
     broadcast_from_rank0: bool = False
     flatten_optimizer_state_dict: bool = False
+    fqn_modifier_name: str = "fqn_modifiers"
```

Comment:
1.  I prefer to make it a private method `_fqn_modifiers` or `_dsd_fqn_modifiers`.
2. Please add to this feature to the document above.